SC_redesign/Crop_and_Soil/soil/carbon_cycling/residue_partition.py

```diff
@@ -60,13 +60,13 @@ def _determine_metabolic_plant_residue_ratio(plant_residue_lignin_composition: f
         Returns
         -------
         float
-            plant lignin to nitrogen ratio (Dmnl)
+            plant lignin to nitrogen ratio (Dimensionless unit)
```

Comment:
I don't like "dimensionless" let's just go with "unitless here, and elsewhere.
```suggestion
            plant lignin to nitrogen ratio (unitless)
```